packages/ui-voip/src/components/VoipPopup/components/VoipPopupContainer.tsx

```diff
@@ -32,18 +32,24 @@ const Container = styled(
 	min-height: 128px;
 	border-radius: 4px;
 	border: 1px solid ${Palette.stroke['stroke-dark'].toString()};
-	box-shadow: 0px 0px 1px 0px ${Palette.shadow['shadow-elevation-2x'].toString()},
+	box-shadow:
+		0px 0px 1px 0px ${Palette.shadow['shadow-elevation-2x'].toString()},
 		0px 0px 12px 0px ${Palette.shadow['shadow-elevation-2y'].toString()};
-	background-color: ${(p) => (p.secondary ? Palette.surface['surface-neutral'].toString() : Palette.surface['surface-light'].toString())};
+	background-color: ${Palette.surface['surface-tint'].toString()};
 	z-index: 100;
 `;
 
-const VoipPopupContainer = ({ children, secondary = false, position = { top: 0, left: 0 }, ...props }: ContainerProps) => (
-	<FocusScope autoFocus restoreFocus>
-		<Container aria-labelledby='voipPopupTitle' secondary={secondary} position={position} {...props}>
-			{children}
-		</Container>
-	</FocusScope>
+const VoipPopupContainer = forwardRef<HTMLDivElement, ContainerProps>(
+	({ children, position = { top: 0, left: 0 }, dragHandleRef, ...props }, ref) => (
+		<FocusScope autoFocus restoreFocus>
+			<Container ref={ref} aria-labelledby='voipPopupTitle' position={position} {...props}>
+				<VoipPopupDragHandle ref={dragHandleRef} />
+				{children}
+			</Container>
+		</FocusScope>
+	),
 );
 
+VoipPopupContainer.displayName = 'VoipPopupContainer';
```

Comment:
Using a named function as the render function in `forwardRef()` does the same trick.